webapp/channels/src/selectors/admin_console.tsx

```diff
@@ -32,14 +32,15 @@ export const getConsoleAccess = createSelector(
     getMySystemPermissions,
     (adminDefinition, mySystemPermissions) => {
         const consoleAccess = {read: {}, write: {}};
-        const addEntriesForKey = (entryKey) => {
+        const addEntriesForKey = (entryKey: string) => {
             const permissions = ResourceToSysConsolePermissionsTable[entryKey].filter((x) => mySystemPermissions.has(x));
-            consoleAccess.read[entryKey] = permissions.length !== 0;
-            consoleAccess.write[entryKey] = permissions.some((permission) => permission.startsWith('sysconsole_write_'));
+            Object.assign(consoleAccess.read, {[entryKey]: permissions.length !== 0});
+            Object.assign(consoleAccess.write, {[entryKey]: permissions.some((permission) => permission.startsWith('sysconsole_write_'))});
```

Comment:
If this is just a workaround for TS complaining when we're adding fields to the empty objects in `consoleAccess`, I think I'd rather type `consoleAccess` as `any` than work around that with `Object.assign` since I don't imagine we'll get an accurate type for `consoleAccess` either way